composer.json

```diff
@@ -19,9 +19,14 @@
 		"symfony/form": "~2.4"
     },
 	"require-dev": {
+        "phpunit/phpunit": "~4.3",
         "phpspec/phpspec": "~2.0",
-        "monolog/monolog": "~1.11"
+        "monolog/monolog": "~1.11",
+        "doctrine/orm": "~2.4.0"
```

Comment:
Any reason that doctrine is required? In the test below you're mocking doctrine.